### DISCUSSION TITLE
Carry over Vercel Preview comments to a changed URL

### DISCUSSION THREAD

**Discussion by kayx23** [vercel/community#1845] [Idea]
### Summary

My team's using vercel preview to review docs and leave review comments. Occasionally there are corrections to the slug in markdown, which results in a change in the URL and loss of all comments.

I understand comments are recovered if we change the URL back -- however, is there a better way? such as "transferring" the comments over? 

TIA!

**Comment by leerob** [comment]
Great feedback, thank you!
